.gitlab-ci.yml

```diff
@@ -97,6 +97,22 @@ before_script:
       - tags
       - ^dca([\d.-]|rc)+$
 
+#
+# Job conditions
+#
+
+# run job only when a new tag is pushed for the cluster agent.
+
+.run_on_cluster_agent_ref: &run_on_cluster_agent_ref
+  only:
+    - /^dca([\d.-]|rc)+$/
```

Comment:
This regex can match any git ref, which includes branches or tags. Gitlab does not support running a job for a specific tag.

Reply:
I this you can. Look at those 2 doc https://docs.gitlab.com/ee/ci/yaml/#only-and-except-simplified and https://docs.gitlab.com/ee/ci/variables/#predefined-variables-environment-variables.

I'm guessing this will do the trick:
```yaml
only:
  refs:
    - tags
  variables:
    - CI_COMMIT_TAG =~ /^dca([\d.-]|rc)+$/
```

I didn't tested it ...

Reply:
That could work. It does say you can use regexes with variables. 👍